### ISSUE TITLE
TaxRateType does not handle blank tax rates

### ISSUE THREAD

**Issue by NyanKiyoshi** [mirumee/saleor#3278]
### What I'm trying to achieve
View product types.

### Steps to reproduce the problem
1. Go to the settings of dashboard 2.0 (reproducible on the demo as well);
2. Open the product types;
3. It should crash (works on clean db + populatedb) [`Expected a value of type "TaxRateType" but received: `].

**System information**
Operating system: ArchLinux 
Browser: Firefox 62.0.2

### Request
```
fragment ProductTypeFragment on ProductType {
    id name hasVariants isShippingRequired taxRate __typename
}
query ProductTypeList($after: String, $before: String, $first: Int, $last: Int) {
    productTypes(after: $after, before: $before, first: $first, last: $last) {
        edges {
            node { ...ProductTypeFragment __typename
            }
            __typename
        }
        pageInfo {
            hasNextPage hasPreviousPage startCursor endCursor __typename
        }
        __typename
    }
}
```

### Variables
```
{"next": 20}
```

### Response
```json
{
  "errors": [
    {
      "message": "Expected a value of type \"TaxRateType\" but received: ",
      "path": [
        "productTypes",
        "edges",
        13,
        "node",
        "taxRate"
      ]
    }
  ],
  "data": {
    "productTypes": {
      "edges": [
        // ...
        {
          "node": {
            "id": "UHJvZHVjdFR5cGU6MTA=",
            "name": "Wine",
            "hasVariants": false,
            "isShippingRequired": true,
            "taxRate": null,
            "__typename": "ProductType"
          },
          "__typename": "ProductTypeCountableEdge"
        }
      ],
      "pageInfo": {
        "hasNextPage": false,
        "hasPreviousPage": false,
        "startCursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "endCursor": "YXJyYXljb25uZWN0aW9uOjEz",
        "__typename": "PageInfo"
      },
      "__typename": "ProductTypeCountableConnection"
    }
  }
}
```

**Comment by NyanKiyoshi** [comment]
The reason of this error is from here:
https://github.com/mirumee/saleor/blob/049dc5c945145cc464cff526074d8fba07919aff/saleor/static/db.json#L93

`tax_rate` can be blank if there is no tax rate associated (I guess that's the reason), the `TaxRateType` enum is not handling the blank value.

**Comment by NyanKiyoshi** [comment]
Resolved in #3340.
